com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/archive/test/ManifestTest.java

```diff
@@ -460,90 +489,9 @@ public void testCreateManifest_Resources() throws NoSuchAlgorithmException, Inva
             }
 
             if (current.getUrl().equals("/main/level1.collectionproxyc")) {
-                assertEquals(5, current.getDependantsCount());
+                printDeps(data, current);
+                assertEquals(4, current.getDependantsCount());
```

Comment:
Old code worked wrong:
root
```
+--/main/main.collectionc
   +--/main/main.goc
   |  +--/main/main.scriptc
   |
   +--/main/shared_go.goc
   |
   +--/main/dynamic.collectionc
   |  +--/main/dynamic.goc
   |
   +--/main/level1.collectionproxyc
      +--/main/level1.collectionc
         +--/main/dynamic.goc
         +--/main/level1.goc
         |  +--/main/level1.scriptc
         |  +--/main/shared_go.goc
         |
         +--/main/level2.collectionproxyc
            +--/main/level2.collectionc
                +--/main/dynamic.goc
                +--/main/level2.goc
                +--/main/level2.soundc
```

new code includes only:
```
    [junit] /main/level1.goc
    [junit] /main/level1.scriptc
    [junit] /main/level2.collectionproxyc
    [junit] /main/level1.collectionc
```

old code includes:
```
    [junit] /main/level1.goc
    [junit] /main/level1.scriptc
    [junit] /main/shared_go.goc
    [junit] /main/level2.collectionproxyc
    [junit] /main/level1.collectionc
```


But `/main/shared_go.goc` used in the main bundle and shouldn't be included.